CONTRIBUTING.md

```diff
@@ -50,6 +50,8 @@ Documentation improvements are always welcome:
 - Follow the existing code style in the repository
 - Include appropriate type definitions
 - Add comments for complex logic
+- Use `uvx ruff check` to check your code
+- Use `uvx ruff format` to automatically format your code 
```

Comment:
We should be clear that this is for Python only

Reply:
Add a disclaimer